rn-test.config.js

```diff
@@ -0,0 +1,9 @@
+'use strict'
```

Comment:
can you move this to a `config` folder ?

Reply:
`lilconfig` won't pick this up that way. :(